2024/migrant-issue-shelters-bar/index.html

```diff
@@ -0,0 +1,42 @@
+<html lang="en">
+    <head>
+        <title>Migrants at Shelters</title>
```

Comment:
This should match the title in the meta_data file and chart, or at least the part of it sans the year appendage on the end.

Reply:
ops missed updating this one, thanks